### DISCUSSION TITLE
2024/11/09/cross-compile-cloudeve-for-loongson-2k0300/

### DISCUSSION THREAD

**Discussion by giscus** [hmsjy2017/blog-comment#10] [Announcements]
# 2024/11/09/cross-compile-cloudeve-for-loongson-2k0300/

前言龙芯有新世界和旧世界之分，两个世界的软件不通用，更多细节请参考这篇文章。本文编译的 Cloudreve 仅适用于龙芯新世界。截至本文撰写时，适配久久派的新世界系统仅有 Buildroot。 提示如果你只是想要一个编译好的可执行文件，可直接前往此处下载。   本文编译所用系统为 Ubuntu 22.04 LTS amd64，其他系统大同小异。以下为详细步骤。 安装 Golang 环境下载 Gol

https://blog.iamsjy.com/2024/11/09/cross-compile-cloudeve-for-loongson-2k0300/

**Comment by Elain-ai** [comment]
大佬您好！我是从这个链接找过来的https://make.quwj.com/project/416              我们来自南京艺术学院！我们很想学习这个项目！请问您是否有他的联系方式

**Reply by hmsjy2017** [reply]
很抱歉，我没有他的联系方式😥

**Reply by Elain-ai** [reply]
好的，谢谢您！
